src/game_inventory.cpp

```diff
@@ -568,7 +568,7 @@ class comestible_inventory_preset : public inventory_selector_preset
 
     protected:
         int get_order( const item_location &loc, const time_duration time ) const {
-            const item &it = const_cast<item &>( *loc );
+            const item &it = static_cast<const item &>( *loc );
```

Comment:
`*loc` already yields a `const item &`, why cast it into the very same thing?

Reply:
I did not really checked what loc is